src/ukrainian-vareniki.md

```diff
@@ -0,0 +1,34 @@
+# Ukrainian Vareniki
+
+(Vareniki/Dumplings) Vareniki are popular all over the world, but aboard they have different names. 
+
+- ⏲️ Prep: 30 mins
+- 🍳 Cook time: 25 mins  
+- 🍽️ Servings: 45
+
+## Ingredients 
+
+- 18 ounces potatoes
+- salt and freshly ground black pepper
+- 3 cups all-purpose flour
+- 1 teaspoon salt
+- ¼ cup sour cream
+- ½ teaspoon baking soda
+- ½ cup water
+- ¼ cup butter
+- 2 onions, finely diced
+- 3 ½ cups finely chopped button mushrooms
+
+## Directions
+
+1. Place potatoes into a large pot and cover with salted water; bring to a boil. Reduce heat to medium-low and simmer until tender, about 25 minutes. Drain potatoes and mash; season with salt and pepper.
+2. Combine flour and 1 teaspoon salt in a large bowl while potatoes are cooking. Stir sour cream and baking soda together in a second bowl. Add water to flour mixture and blend in, followed by sour cream mixture. Knead into a smooth dough, cover with plastic wrap, and refrigerate for 10 to 15 minutes.
+3. Melt butter in a skillet over medium heat and cook onions until browned, 5 to 10 minutes. Add mushrooms and cook until browned, about 5 minutes. Add to mashed potatoes.
+4. Divide dough into 4 portions and roll each portion into a 1 1/2-inch thick rope. Cut ropes into small 1/2-inch pieces. Press each dough piece into a small flat circle and place 1 teaspoon of potato filling in the center. Fold dough over into a half-moon shape and seal edges.
+5. Bring a large pot of salted water to a gentle boil. Add the vareniki in batches and cook until they float to the surface, about 3 minutes. Cook for 2 to 3 minutes more. Remove with a slotted spoon and place in a colander to drain.
+
+## Contributors
+
+- [Vlad](https://github.com/VladPetryshyn)
+
+;tags: ukrinian vareniki dumplings potato
```

Comment:
```suggestion
;tags: ukrainian dumpling potato
```

`vareniki` is too specific in my opinion. Anyone looking for such a recipe would be better off probably searching the page.